packages/host/src/lib/csi-controller.ts

```diff
@@ -672,7 +672,15 @@ export class CSIController extends TypedEmitter<Events> {
             }
 
             const out = new DataStream();
-            const handler = (data: any) => out.write(data);
+            const handler = (data: any) => {
+                if (typeof data !== "object") {
+                    out.write(data);
+                }
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                const { _eventName, ...message } = data;
```

Comment:
there is no `_eventName` in event message payload so this assignment will cause `message` to be `data`.